etstool.py

```diff
@@ -98,46 +98,61 @@
 # The requirement is to be interpreted by EDM
 TRAITS_VERSION_REQUIRES = os.environ.get("TRAITS_REQUIRES", "")
 
+# Required runtime dependencies. Should match install_requires in setup.py
 dependencies = {
-    "numpy",
     "traits" + TRAITS_VERSION_REQUIRES,
-    "pandas",
-    "pygments",
-    "pip",
-    "nose",
-    "coverage",
-    "configobj",
-    "docutils",
+    # pyface is installed from source via ci-src-requirements.txt
 }
 
 # NOTE : pyface is always installed from source
 source_dependencies = {
     "traits": "git+http://github.com/enthought/traits.git#egg=traits",
 }
 
-# Additional toolkit-independent dependencies for demo testing
-# This should correspond to the "examples" dependencies in extras_requires but
-# the following names are used by EDM.
-test_dependencies = {
-    "apptools",
-    "chaco",
-    "h5py",
-    "numpy",
-    "pandas",
-    "pytables",
-}
-
+# The following should match extras_require in setup.py but with package
+# names compatible with EDM
 extra_dependencies = {
     # XXX once pyside2 is available in EDM, we will want it here
-    'pyside2': set(),
-    'pyqt': {'pyqt<4.12'},  # FIXME: build 1 of 4.12.1 appears to be bad
-    'pyqt5': {'pyqt5'},
+    'pyside2': {
+        'pygments',
+    },
+    'pyqt': {
+        'pyqt<4.12',  # FIXME: build 1 of 4.12.1 appears to be bad
+        'pygments',
+    },
+    'pyqt5': {
+        'pyqt5',
+        'pygments',
+    },
     # XXX once wxPython 4 is available in EDM, we will want it here
-    'wx': set(),
-    'null': set()
+    'wx': {
+        'numpy',
```

Comment:
Not sure why `numpy` is a dependency of `wx` toolkit. I understand that this is emulating what we see in `setup.py` but still.

Reply:
From what is covered by tests, numpy is required for `traitsui.theme` via `traitsui.wx.image_slice`.
I guess there are more... e.g. I can find `numpy` being imported at the top-level by `pyface.wx.color`.
So at the moment, it does not seem at all disposable for wx.

Reply:
ohh yeah. makes sense. 👍